### PR TITLE
removed bundle ID from console output for error handling

### PR DESCRIPTION
**Pull request by willsaid-google**
This removes this:

Screen Shot 2020-05-20 at 4 37 14 PM

### CODE REVIEW

File: CommonLib/Error/GREYError+Private.h [hunk 1]
```diff
@@ -59,16 +59,6 @@ GREY_EXTERN NSString *const kErrorFunctionNameKey;
  */
 GREY_EXTERN NSString *const kErrorUserInfoKey;
 
-/**
- * Key used to retrieve the error info from an error object dictionary.
- */
-GREY_EXTERN NSString *const kErrorErrorInfoKey;
-
```
Comment (tirodkar) [inline CommonLib/Error/GREYError+Private.h:66]: this is extra?

Comment (willsaid-google) [reply CommonLib/Error/GREYError+Private.h:66]: Fixed

**Comment by tirodkar** [comment]
@chan-park is this fine?

**Comment by chan-park** [comment]
LGTM

**Comment by willsaid-google** [comment]
@chan-park the checks were not successful because of 
```
`ErrorHandlingTest`
  testExceptionDetailsForAMatcherFailure, (((int)screenshots.count) equal to (2)) failed: ("5") is not equal to ("2")
```

Can we ignore?

**Pull request closed without merging**